x/conflict/keeper/conflict.go

```diff
@@ -20,6 +20,11 @@ func (k Keeper) ValidateFinalizationConflict(ctx sdk.Context, conflictData *type
 }
 
 func (k Keeper) ValidateResponseConflict(ctx sdk.Context, conflictData *types.ResponseConflict, clientAddr sdk.AccAddress) error {
+	// 0. validate conflictData is not nil
+	if conflictData.IsDataNil() {
```

Comment:
can this be also in the validate basic?
also if we check it here we dont have to check it anymore, the rest are redundant

Reply:
done. see https://github.com/lavanet/lava/commit/bfaa18c0969b329cfb0d19481615c5f0cec66cfe